src/AssignmentsStorage.php

```diff
@@ -120,39 +125,39 @@ public function hasItem(string $name): bool
 
     public function renameItem(string $oldName, string $newName): void
     {
-        if ($oldName === $newName) {
-            return;
-        }
-        $this->database
-            ->update($this->tableName, ['itemName' => $newName], ['itemName' => $oldName])
-            ->run();
+        $message = 'Use "ItemStorage::update()" instead, all related assignments will be updated automatically.';
+        throw new RuntimeException($message);
```

Comment:
`renameItem()` call in `Manager`: [1](https://github.com/yiisoft/rbac/blob/7fcfec40ec749f98a17f02ef4411334166410555/src/Manager.php#L438) [2](https://github.com/yiisoft/rbac/blob/7fcfec40ec749f98a17f02ef4411334166410555/src/Manager.php#L478)

It's should not throw exception.